src/extension/netcdf-out/src/main/java/org/geoserver/web/netcdf/layer/NetCDFOutSettingsEditor.html

```diff
@@ -50,9 +50,15 @@
                     </label>
                 </li>
                 <li class="choiceItem">
-                    <input id="copyAttributes" type="checkbox" wicket:id="copyAttributes" />
-                    <label for="copyAttributes" class="checkbox">
-                        <wicket:message key="NetCDFOut.copyAttributes">Copy Attributes</wicket:message>
+                    <input id="copyVariableAttributes" type="checkbox" wicket:id="copyVariableAttributes" />
```

Comment:
Nice. You found the inexplicable duplicate HTML template files. Very thorough. As mentioned earlier, all references to `copyAttributes` should be left as is for backwards compatibility and consistency.